src/treewidth.jl

```diff
@@ -0,0 +1,166 @@
+using TreeWidthSolver
```

Comment:
Please move all using statements to the module file.

Reply:
The statement has been moved.